.dockerignore

```diff
@@ -1,3 +1,6 @@
 **/.git/
+**/bin/
```

Comment:
@marienfressinaud Do you need that when using `make` in your local Docker image?

Reply:
I don't think so since I'm mounting the root directory in the container